src/sfctl/tests/request_generation_test.py

```diff
@@ -331,10 +331,10 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements
             '/Nodes/nodeName',
             ['api-version=6.0'])
         self.validate_command(  # list
-            'sfctl node list --continuation-token=nodeId --node-status-filter=up',
+            'sfctl node list --continuation-token=nodeId --node-status-filter=up --max-results=3',
             'GET',
             '/Nodes',
-            ['api-version=6.0', 'ContinuationToken=nodeId', 'NodeStatusFilter=up'])
+            ['api-version=6.3', 'ContinuationToken=nodeId', 'NodeStatusFilter=up', 'MaxResults=10'])
```

Comment:
Why is it MaxResults=10 and --max-results=3 above? Do they have to match? I can't remember how the validate_command works.

Reply:
Typo, thanks :)